esp-hal/src/lcd_cam/cam.rs

```diff
@@ -141,6 +144,8 @@ impl<'d> Camera<'d> {
         CH: DmaChannelConvert<<LCD_CAM as DmaEligible>::Dma>,
         P: RxPins,
     {
+        let guard = PeripheralGuard::new(crate::system::Peripheral::LcdCam);
```

Comment:
Could this just be moved out of `Cam`?

Reply:
Ah I see - you need a `Cam` to construct a `Camera`